charts/em-hrs-api/values.yaml

```diff
@@ -18,10 +18,6 @@ java:
           alias: STORAGEACCOUNT_PRIMARY_CONNECTION_STRING
         - name: cvp-storage-connection-string
           alias: CVP_STORAGE_CONNECTION_STRING
-        - name: cvp-storage-container-name
-          alias: CVP_STORAGE_CONTAINER_NAME
```

Comment:
Removing this will cause issue. Please update the correct name here as well - https://github.com/hmcts/em-hrs-api/blob/master/charts/em-hrs-api/values.yaml#L52